src/models/configs.ts

```diff
@@ -49,6 +49,7 @@ export interface ReportPortalConfig extends ClientConfig, AttachmentsConfig {
   rerun?: boolean;
   rerunOf?: string;
   mode?: LAUNCH_MODES;
+  reportLastErrorLog?: boolean;
```

Comment:
Such name sounds as "report the last error log or not" and doesn't connected with extending description with it, but the last error log will be reported anyway.
I suggest to rename it to smth like `extendTestDescriptionWithLastError`.

Reply:
The biggest problem with this name is its length, but it is incredibly descriptive!